src/main/java/org/concordion/api/CommandCall.java

```diff
@@ -90,4 +90,18 @@ public String getParameter(String camelCaseParameterName, String spinalCaseParam
         }
         return null;
     }
+
+    public boolean shouldExecuteWithNonExampleChildren() {

```

Comment:
I'm having trouble reconciling the method names - this one has `withNonExampleChildren`, the method it calls has `evenIfNoNonExampleChildren`. Can we simplify it - removing the double negative?